src/components/FriendCard/friendcards.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState, useEffect } from "react";
+import Parse from "parse/dist/parse.min.js";
+import { Link } from "react-router-dom";
+import "./friendcards.css";
+import ProfilePicture from "../shared/ProfilePicture/ProfilePicture";
+
+function FriendCards({
+	filteredUsers,
+	allFriends,
+	currentUserName,
+	//addedUsers
+}) {
+	return (
+		<div>
+			{filteredUsers.map((user, index) =>
+				user ? (
+					allFriends?.some((friend) => friend.username === user.username) ? (
+						<Card
+							key={index}
+							username={user.username}
+							userid={user.id}
+							profilepicture={user.profilepicture}
+							currentUserName={currentUserName}
+							isFriend={true}
+							//isAdded={addedUsers?.some(addedUser => addedUser.username === user.username) ? true : false}
+						/>
```

Comment:
Consider removing commented-out parts of your code before the exam. Also, make sure that your code is formatted in a consistent way :)